components/dropdown/dropdown.test.js

```diff
@@ -36,6 +36,7 @@ describe('DtDropdown Tests', () => {
   let wrapper;
   let anchorElement;
   let listWrapper;
+  let buttonExists;
```

Comment:
```suggestion
  let closeButton;
```

`buttonExists` doesn't make sense.

Reply:
👍🏼 , will update the name to `closeButton`